### PR TITLE
refactor: clean up spreadsheet ApplicationConnection

### PR DESCRIPTION
**Pull request by tomivirkki**
Part of #3119 

Remove some unused code from ApplicationConnection.

Shrinks the `spreadsheet-export.js` bundle size from 1.4MB to 1.2MB

### CODE REVIEW

**Comment by manolo** [comment]
Good clean up, it reduced ~15% the final size of the bundle

Screenshot 2022-06-02 at 17 22 27

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java [hunk 1]
```diff
@@ -474,59 +369,13 @@ public void start() {
...
      */
     private boolean isActive() {
-        return !getMessageHandler().isInitialUidlHandled() || isWorkPending()
-                || getMessageSender().hasActiveRequest()
-                || isExecutingDeferredCommands();
+        return false;
     }
 
     private native void initializeTestbenchHooks(
             ComponentLocator componentLocator, String ttAppId)
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java:376]: Can't this method be removed? Only reference to it seems to be on the `init` method.

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java [hunk 2]
```diff
@@ -561,28 +410,7 @@ private JavaScriptObject getVersionInfo() {
      */
     private native void initializeClientHooks()
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java:411]: Similar to `initializeTestbenchHooks`.

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java [hunk 3]
```diff
@@ -760,7 +582,7 @@ private boolean isExecutingDeferredCommands() {
      * @return The loading indicator for this ApplicationConnection
      */
     public VLoadingIndicator getLoadingIndicator() {
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java:584]: It seems that both methods can be removed. Couldn't find them being used anywhere.

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java [hunk 4]
```diff
@@ -772,26 +594,19 @@ public VLoadingIndicator getLoadingIndicator() {
...
     private void addVariableToQueue(String connectorId, String variableName,
             Object value, boolean immediate) {
-        boolean lastOnly = !immediate;
-        // note that type is now deduced from value
-        serverRpcQueue.add(new LegacyChangeVariablesInvocation(connectorId,
-                variableName, value), lastOnly);
-        if (immediate) {
-            serverRpcQueue.flush();
-        }
     }
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java:602]: It seems that this method can be removed together with all `updateVariable` methods.

**Review by DiegoCardoso — CHANGES_REQUESTED** [review]

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java (hunk 1 above)
Comment (tomivirkki) [reply vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java]: Done

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java (hunk 3 above)
Comment (tomivirkki) [reply vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java:475]: Screenshot 2022-06-07 at 15 39 35

**Review by DiegoCardoso — APPROVED** [review]
Nice cut! 🎉

**Pull request merged**